### PR TITLE
Feat 15 게시물 등록

### PR DESCRIPTION
**Pull request by EunjungOh123**

### CODE REVIEW

**Comment by naekang** [comment]
|File|Coverage [52.14%]|:x:|
|:-|:-:|:-:|
|[PostController.java](https://github.com/Gaethering/Gaethering-Server/blob/63eed251f5c20dce311e6b7a469be8a11fcf11ad/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fcontroller%2FPostController.java)|100%|:green_apple:|
|[PostServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/63eed251f5c20dce311e6b7a469be8a11fcf11ad/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fservice%2FPostServiceImpl.java)|79.85%|:x:|
|[Post.java](https://github.com/Gaethering/Gaethering-Server/blob/63eed251f5c20dce311e6b7a469be8a11fcf11ad/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fdomain%2FPost.java)|62.94%|:x:|
|[PostImage.java](https://github.com/Gaethering/Gaethering-Server/blob/63eed251f5c20dce311e6b7a469be8a11fcf11ad/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fdomain%2FPostImage.java)|3.23%|:x:|
|[PostErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/63eed251f5c20dce311e6b7a469be8a11fcf11ad/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fexception%2FerrorCode%2FPostErrorCode.java)|0%|:x:|

|Total Project Coverage|53.41%|:x:|
|:-|:-:|:-:|

**Comment by naekang** [comment]
|File|Coverage [52.14%]|:x:|
|:-|:-:|:-:|
|[PostController.java](https://github.com/Gaethering/Gaethering-Server/blob/0e1273945140a5c68043c190b74154bdfeb5fbea/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fcontroller%2FPostController.java)|100%|:green_apple:|
|[PostServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/0e1273945140a5c68043c190b74154bdfeb5fbea/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fservice%2FPostServiceImpl.java)|79.85%|:x:|
|[Post.java](https://github.com/Gaethering/Gaethering-Server/blob/0e1273945140a5c68043c190b74154bdfeb5fbea/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fdomain%2FPost.java)|62.94%|:x:|
|[PostImage.java](https://github.com/Gaethering/Gaethering-Server/blob/0e1273945140a5c68043c190b74154bdfeb5fbea/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fdomain%2FPostImage.java)|3.23%|:x:|
|[PostErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/0e1273945140a5c68043c190b74154bdfeb5fbea/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fexception%2FerrorCode%2FPostErrorCode.java)|0%|:x:|

|Total Project Coverage|53.75%|:x:|
|:-|:-:|:-:|

**Comment by naekang** [comment]
|File|Coverage [52.14%]|:x:|
|:-|:-:|:-:|
|[PostController.java](https://github.com/Gaethering/Gaethering-Server/blob/e490d07a320482dbf1742ad203b6da3196fc099b/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fcontroller%2FPostController.java)|100%|:green_apple:|
|[PostServiceImpl.java](https://github.com/Gaethering/Gaethering-Server/blob/e490d07a320482dbf1742ad203b6da3196fc099b/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fservice%2FPostServiceImpl.java)|79.85%|:x:|
|[Post.java](https://github.com/Gaethering/Gaethering-Server/blob/e490d07a320482dbf1742ad203b6da3196fc099b/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fdomain%2FPost.java)|62.94%|:x:|
|[PostImage.java](https://github.com/Gaethering/Gaethering-Server/blob/e490d07a320482dbf1742ad203b6da3196fc099b/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fdomain%2FPostImage.java)|3.23%|:x:|
|[PostErrorCode.java](https://github.com/Gaethering/Gaethering-Server/blob/e490d07a320482dbf1742ad203b6da3196fc099b/src%2Fmain%2Fjava%2Fcom%2Fgaethering%2Fgaetheringserver%2Fboard%2Fexception%2FerrorCode%2FPostErrorCode.java)|0%|:x:|

|Total Project Coverage|53.75%|:x:|
|:-|:-:|:-:|

File: src/main/java/com/gaethering/gaetheringserver/board/dto/PostResponse.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PostResponse {
+
+    private String title;
+
+    private String content;
+
+    private List<String> imgUrlList;
```
Comment (naekang) [inline src/main/java/com/gaethering/gaetheringserver/board/dto/PostResponse.java:22]: 변수명을 `imageUrls`로 수정하는건 어떨까요?

File: src/main/java/com/gaethering/gaetheringserver/board/dto/PostRequest.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PostRequest {
```
Comment (naekang) [inline src/main/java/com/gaethering/gaetheringserver/board/dto/PostRequest.java:13]: Request validation도 추가해주시면 좋을거같습니다!

**Review by naekang — COMMENTED** [review]
게시글 쪽 엄청 빠르게 작성해주셨네요 수고하셨습니다!! 😀😀😀

++ 코드 정렬이 각자 다 달라서 crtl + alt + l 이용해서 정렬한번 해주시고 올려주시면 좀 더 통일성있게 올라올거같아요!

File: src/main/java/com/gaethering/gaetheringserver/board/domain/Post.java [hunk 3]
```diff
@@ -25,7 +24,7 @@
 @Getter
 @Builder
 @AllArgsConstructor
-@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@NoArgsConstructor
```
[the remark this answers is not in the document]
Comment (EunjungOh123) [reply src/main/java/com/gaethering/gaetheringserver/board/domain/Post.java:29]: 아 다들 안 쓰시는 거 같아서 뺐었는데 다시 넣도록 하겠습니다

**Pull request closed without merging**